wagtailsharing/tests/test_urls.py

```diff
@@ -1,20 +1,20 @@
-from __future__ import absolute_import, unicode_literals
+from django.test import TestCase
+
+import wagtail.core.urls as wagtail_core_urls
+import wagtailsharing.urls
+from mock import patch
+
 
 try:
     from importlib import reload
 except ImportError:
     pass
```

Comment:
`importlib.reload` is available from 3.4 on, so this `try`/`except` can be removed.

```suggestion
from importlib import reload
```